frontend/src/components/releaseEditor/ReviewModal.vue

```diff
@@ -13,6 +13,7 @@
     >
       <template #body>
         <div class="alert alert-danger mb-4" role="alert">
+          <!-- FIXME: is there a way to deduplicate this + reminders.jinja -->
```

Comment:
easiest way would be mounting a vue component in place of the template. Feels silly doing so for static markup but may beat duplicating it

Reply:
I suspected that might be the only way forward since we regularly embed frontend things into the backend but not vice versa easily.. I think it's not a blocker for this PR moving forward, am OK with having two sets of guidance, I'll make an issue out of it maybe for a student dev to take on